specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json

```diff
@@ -2163,6 +2263,47 @@
         }
       }
     },
+    "EndpointDependency": {
+      "description": "A domain name that a service is reached at, including details of the current connection status.",
+      "type": "object",
+      "properties": {
+        "domainName": {
+          "description": "The Domain Name of the dependency.",
```

Comment:
Nit - inconsistent casing of "Domain Name" in the EndpointDependency description vs domainName property description